src/Tickets/Commerce/Gateways/Stripe/Webhooks/Charge_Webhook.php

```diff
@@ -25,6 +27,28 @@ public static function handle( array $event, Status_Interface $new_status, \WP_R
 		$order = tribe( Order::class )->get_from_gateway_order_id( $payment_intent_id );
 
 		if ( empty( $order ) ) {
+
+			if ( empty( $charge_data['metadata'][ Payment_Intent::$tc_metadata_identifier ] ) ) {
+				$response->set_status( 200 );
+				$response->set_data( sprintf(
+				// Translators: %1$s is the event id and %2$s is the event type name.
+					__( 'Event %1$s was received and will not be handled because the Payment Intent %2$s does not refer to an Event Tickets transaction.', 'event-tickets' ),
+					esc_html( Arr::get( $event, 'id' ) ),
+					esc_html( $payment_intent_id )
+				) );
+
+				return $response;
```

Comment:
If we return a `200` response here, will this allow other plugins to respond? I thought we should avoid responding with a `return;` only.

Reply:
Thanks for the clarification, sounds good 🚀